### DISCUSSION TITLE
Create SimpleX group

### DISCUSSION THREAD

**Discussion by ghost** [minibolt-guide/ramix-node#1] [Ideas]
Hello, I am very very pleased to see this fork as raspibolt guide seem a bit abandoned. I would like to suggest the creation of a simpleX group as many nostriches are already there, as telegram is total no go zone.

**Comment by twofaktor** [comment]
We already have a Nostr channel:

https://www.nostrchat.io/channel/aa64f2ead929ce8417f85bde7d22ebde13cc01ceb4e00145572437eb1ad46249

Maintaining too many communication channels is difficult. It is not planned at the moment. Sorry 🙏
